### PR TITLE
CRYP-23: Implement signout button and functionality on settings page

### PR DESCRIPTION
**Pull request by PolaFarid**
unknown

### CODE REVIEW

**Review by jason-gerard — APPROVED** [review]
Looks good, nice work.

File: packages/client/theme/components/button.ts [hunk 1]
```diff
@@ -24,6 +24,19 @@ export const button = {
                 opacity: "60",
             },
         },
+        outline: {
+            bg: "white",
+            borderColor: "text.200",
+            _text: {
+                color: "error.500",
```
Comment (domsec) [inline packages/client/theme/components/button.ts:31]: Set the default text color "text.700" for outline variant buttons. Specific text colors should be set per Button component instance.

File: packages/client/components/SignOutButton.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    async function handleSignOut(): Promise<void> {
+        // TODO refactor this function to redirect user to welcome screen
+        // this will be done automatically when the token is removed
+        // after refactoring the navigation which will be done in Sprint 3
+        // when the entire sign up / sign in flow is completed
+        await StorageService.remove(KEY_JWT);
+    }
+
+    return (
+        <Button variant="outline" _text={styles.signOutButtonText} onPress={handleSignOut}>
```
Comment (domsec) [inline packages/client/components/SignOutButton.tsx:17]: Replace `_text={styles.signOutButtonText}` with `style={styles.signOutButton}` and add

```
_text={{
      color: "error.500"
    }}
```

**Review by domsec — COMMENTED** [review]
Re-submit a PR with these changes.

File: packages/client/components/SignOutButton.tsx (hunk 2 above)
Comment (Andre-Ibrahim) [reply packages/client/components/SignOutButton.tsx:17]: Button style prop doesnt take a StyleSheet object like that

Comment (Andre-Ibrahim) [reply packages/client/components/SignOutButton.tsx:17]: `Type '{ fontSize: number; lineHeight: number; }' is not assignable to type 'StyleProp<ViewStyle> | ((state: PressableStateCallbackType) => StyleProp<ViewStyle>)'.ts(2322)
`

**Pull request merged**